test/uk/gov/hmrc/transitmovementsconverter/models/testobjects/CC007CTestMessageType.scala

```diff
@@ -0,0 +1,281 @@
+      |                    "eMailAddress": "string"
+      |                }
+      |            },
+      |            "Incident": [
+      |                {
+      |                    "sequenceNumber": "token",
+      |                    "code": "token",
+      |                    "text": "string",
+      |                    "Endorsement": {
+      |                        "date": "2013-05-22+01:00",
+      |                        "authority": "string",
+      |                        "place": "string",
+      |                        "country": "st"
+      |                    },
+      |                    "Location": {
+      |                        "qualifierOfIdentification": "token",
+      |                        "UNLocode": "token",
+      |                        "country": "st",
+      |                        "GNSS": {
+      |                            "latitude": "string",
+      |                            "longitude": "string"
+      |                        },
+      |                        "Address": {
+      |                            "streetAndNumber": "string",
+      |                            "postcode": "string",
+      |                            "city": "string"
+      |                        }
+      |                    },
+      |                    "TransportEquipment": [
+      |                        {
+      |                            "sequenceNumber": "token",
+      |                            "containerIdentificationNumber": "string",
+      |                            "numberOfSeals": 100,
+      |                            "Seal": [
+      |                                {
+      |                                    "sequenceNumber": "token",
+      |                                    "identifier": "string"
+      |                                }
+      |                            ],
+      |                            "GoodsReference": [
+      |                                {
+      |                                    "sequenceNumber": "token",
+      |                                    "declarationGoodsItemNumber": 100
+      |                                }
+      |                            ]
+      |                        }
+      |                    ],
+      |                    "Transhipment": {
+      |                        "containerIndicator": "0",
+      |                        "TransportMeans": {
+      |                            "typeOfIdentification": "token",
+      |                            "identificationNumber": "string",
+      |                            "nationality": "st"
+      |                        }
+      |                    }
+      |                }
+      |            ]
+      |        },
+      |        "messageIdentification": "token",
+      |        "CustomsOfficeOfDestinationActual": {
+      |            "referenceNumber": "stringst"
+      |        }
+      |    }
+      |}
+      |""".stripMargin
+  )
```

Comment:
minor, is it possible to get xml and json from a file

Reply:
what is the advantage in doing that?

Reply:
easy to read, can be reused these files if required in other places

Reply:
If I was starting this again, I wouldn't mind do that instead I did what was easiest at the time when I was writing the converter and I didn't want to start writing the file handling in. However, this is a little embedded and would take a bit of time to properly extract.

If this is what we want to do (and I'm happy for that to be the case), a tech debt ticket should be raised so we can properly capture it.

Reply:
yes make sense